docs/pages/functionality/configuration-management.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Configuration Management
+permalink: functionality/configuration-management
+type: functionality
+language: en
+list: include
+---
+
+
+Meshery provides the ability for you as a service mesh manager to customize your service mesh deployment.
+
+#### Notifications
+
+Meshery tracks operations that you perform on service meshes and their workloads. Meshery provides notification of environment issues, application conflicts with service mesh configuration.
+
+#### Acknowledging and dismissing notifications
+
+Many notifications are informational and can readily be dismissed after reading. Some notifications include actionable alerts.
+
+#### Taking action on alerts
+
+Particular notifications that Meshery presents are immediately actionable. These are denoted by a red colored highlight. Be on the lookout for these alerts and take action promptly to ensure a smooth running service mesh.
```

Comment:
@josephmidura How about adding a Screenshot of the Alert so that users can easily know what a red coloured highlight in Meshery looks like?

CC @leecalcote @warunicorn19